modules/microshift-nw-config-router-admission.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-router-admission_{context}"]
+= Configuring router admission policy
+
+{microshift-short} by default, does not allow routes in multiple namespaces to use the same hostname which is changed here to accommodate different use cases. You can use the below router configuration option to disable the `namespaceOwnership` checks which supports combining routes from multiple namespaces.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` file parameters in the `ingress.routerAdmissionPolicy.namespaceOwnership` field to `InterNamespaceAllowed` as shown in the following example:
```

Comment:
I think we should mention the other value for this too, which is `Strict`.

Reply:
The procedure explains about the route configuration option to disable the checks which supports combining routes from multiple namespaces.
How about I  add a note below about `strict` ?